Lib/fontTools/ttLib/sfnt.py

```diff
@@ -172,16 +168,16 @@ def __setstate__(self, state):
 }
 
 
-def compress(data, level=ZLIB_COMPRESSION_LEVEL):
-	""" Compress 'data' to Zlib format. If 'USE_ZOPFLI' variable is True,
+def compress(data, use_zopfli=False, level=ZLIB_COMPRESSION_LEVEL):
```

Comment:
Putting the new parameter before an existing parameter is not backwards-compatible. Previously, `compress(data, 9)` would compress with the best compression level of the module selected by `USE_ZOPFLI`. With this change, the same code will compress with the default compression level of Zopfli.

It is also backwards-incompatible because the current documentation says that setting `USE_ZOPFLI` has an effect, which with this change is no longer true. It would be better to keep `USE_ZOPFLI` but deprecate it and say that the new `use_zopfli` parameter takes precedence if set.

Reply:
Thanks David!  

I attempted to address these issues in 60b47cd244679bb2c015e4f199af6a2b74dde482